crates/mun_codegen/src/lib.rs

```diff
@@ -4,15 +4,20 @@ mod db;
 #[macro_use]
 mod ir;
 
+#[macro_use]
+extern crate mun_codegen_macros;
```

Comment:
Rust has already removed the need for this syntax, now allowing you to use syntax similar to `use some_module::some_fn;` in the file where you use the macro.